winui-toc.html

```diff
@@ -531,6 +531,16 @@
        <li><a href="/winui/SegmentedControl/Disable-Items">Disable Items</a></li>
    </ul>
 </li>
+<li>
+    SfShimmer Control
+   <ul>
+       <li><a href="/winui/SfShimmerControl/Overview">Overview</a></li>
```

Comment:
@GilbertmanojSF3971 Check and change the path properly. change the folder name to Shimmer instead of SfShimmer.

Reply:
@Ganesh-Mariappan Changed the path and named the SfShimmer into Shimmer for the folder